README.rst

```diff
@@ -155,6 +155,36 @@ The available options for the ``wger`` command (if installed from PyPI) or
   migrate_db              Run all database migrations
   start_wger              Start the application using django's built in webserver
 
+API
+===
+To use the API provided by the application, navigate to the `About this Software` part of the application then to `REST API`.
+The API provides endpoints that cab be used to access the different functonalities of the app. This is important as it can be
+consumed to by third party applications/parties if they need to.
+
+USER ENDPOINT
+-------------
+The use endpoint is important since external apps might need to register accounts on behalf of their
+users without having to send them over to the wger instance.
+
+To access the user endpoint got to:
+::
+
+http://localhost:8000/api/v2/users/
```

Comment:
You want to avoid placing the exact local port. Just put a placeholder like http://<localport>/api/v2/users

Reply:
Ok. Working on it.